src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java

```diff
@@ -565,15 +565,20 @@ private void discoverCompostableItems()
     }
 
     /**
-     * Create complete list of compostable items.
+     * Create complete list of plantable items.
      */
     private void discoverPlantables()
     {
         if (plantables.isEmpty())
         {
-            plantables.addAll(ImmutableList.copyOf(allBlocks.stream().filter(this::isPlantable).map(ItemStorage::new).collect(Collectors.toList())));
```

Comment:
Can't we actually put this in the isPlantable check? Would be much more efficient.